pkg/model/ordered_dict.go

```diff
@@ -97,21 +103,20 @@ func CleanDictionary(dict interface{}) Dictionary {
 
 func CleanDictionarySlice(dictSlice interface{}) []Dictionary {
 	result := []Dictionary{}
-
 	switch typedInter := dictSlice.(type) {
 	case []interface{}:
 		for _, d := range typedInter {
-			result = append(result, CleanDictionary(d))
+			result = append(result, CleanDictionary(CleanTypes(d)))
```

Comment:
I don't understand why you need to call CleanTypes(d) here because CleanDictionary already calls CleanTypes by itself:

```go
// CleanDictionary fixes all types in the structure
func CleanDictionary(dict interface{}) Dictionary {
	return CleanTypes(dict).(Dictionary)
}
```

This code trigger CleanTypes twice then (performance issue).

Reply:
You are right, the main change is in CleanTypes, it was the case `map[string]interface{}` that was missing
```go
	case map[string]interface{}:
		dict := NewDictionary()
		for k, v := range typedInter {
			dict.Set(k, CleanTypes(v))
		}
		return dict
```